egs/fame/v2/utils

```diff
@@ -0,0 +1 @@
+../v1/utils
```

Comment:
I think this should point to the original utils dir (I think in WSJ)

Reply:
The same as above..